### DISCUSSION TITLE
Video preview and thumbnails

### DISCUSSION THREAD

**Discussion by wisemonkey** [bpatrik/pigallery2#672] [Q&A]
Does it help if GPU + video encoder is available for ffmpeg?
Mainly for video previews and thumbnails.

If yes do I just install GPU drivers and ffmpeg takes care of everything?

**Comment by bpatrik** [comment]
For thumbnail I do not think it makes a difference.

I cant give guidance on setting it up. Never did myself. Some people did on this forum. You can try to find those threads.
